tests/test_evaluator.py

```diff
@@ -1 +1,162 @@
 """Unit tests for the `evaluator` module."""
+
+from collections import defaultdict
+from tabnanny import verbose
+from typing import Dict
+
+import pytest
+
+from aiai_eval.utils import Device
+from src.aiai_eval.evaluator import Evaluator
+from src.aiai_eval.exceptions import (
+    InvalidArchitectureForTask,
+    ModelDoesNotExistOnHuggingFaceHub,
+)
+from src.aiai_eval.task_configs import NER, SENT
+from src.aiai_eval.task_factory import TaskFactory
+
+
+class TestEvaluator:
+    @pytest.fixture(scope="module")
+    def evaluator(self):
```

Comment:
This is subjective, but I think that setting the `testing=True` attribute here makes a bit more sense than adding it to the individual tests.